src/screens/ProfileScreen/index.js

```diff
@@ -13,12 +15,11 @@ import {
   TouchableNativeFeedback,
   View
 } from 'react-native';
-import PropTypes from 'prop-types';
 import {SafeAreaProvider} from 'react-native-safe-area-context';
 import {debounce} from 'lodash';
 import {showMessage} from 'react-native-flash-message';
 import {useNavigation} from '@react-navigation/core';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `netInfo` has been moved to a different location in the file. This change doesn't affect the functionality of the code, but it's good to keep imports organized for better readability and maintainability.

```diff
- import netInfo from '@react-native-community/netinfo';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->